README.md

```diff
@@ -139,21 +150,16 @@ To run the HTML tests:
 2. In the Solution Explorer, select the Microsoft.WindowsAzure.Mobile.WinJS.Test project and right-click to select 'View in Browser'.
 3. The default browser will launch and run the test HTML application. Some tests may fail because due to an 'Unexpected connection failure'. This is because the test is configured to connect to a Mobile Service that does not exist. These failures can be ignored.
 
-## Additional Resources
+## Useful Resources
 
-* [Quickstarts](https://github.com/Azure/azure-mobile-services-quickstarts)
+* [Quickstarts on Github](https://github.com/Azure/azure-mobile-services-quickstarts)
```

Comment:
i wanted to differentiate them with the quickstart tutorials up above. but it's not completely necessary.